packages/specification/prescriptions-api.yaml

```diff
@@ -0,0 +1,269 @@
+    bundle-response:
+      type: object
+      required:
+        - entry
+        - resourceType
+        - type
+      description: A FHIR searchset Bundle.
+      properties:
+        resourceType:
+          type: string
+          description: FHIR resource type.
+          default: Bundle
+        id:
+          type: string
+          description: Object ID for the Bundle.
+          example: 164996aa-29eb-4f14-a5ac-4b556a7baf6e
+        meta:
+          type: object
+          required: [lastUpdated]
+          properties:
+            lastUpdated:
+              type: string
+              description: The start date of a prescription. Helps the patient in knowing when the prescription is valid from.
+              format: date-time
+              example: "2023-06-05T10:11:12+00:00"
+        type:
+          type: string
+          description: Denotes that the bundle is a list of resources returned as a result of a search.
+          enum: [searchset]
+        entry:
+          type: array
+          description: A collection of resources contained within the Bundle.
+          items:
+            type: object
+            required:
+              - resourceType
+              - identifier
+              - type
+              - entry
+            description: A FHIR collection Bundle.
+            properties:
+              resourceType:
+                type: string
+                description: FHIR resource type.
+                default: Bundle
+              id:
+                type: string
+                description: Object ID for the Bundle.
+                example: 164996aa-29eb-4f14-a5ac-4b556a7baf6e
+              identifier:
+                type: object
+                properties:
+                  system:
+                    type: string
+                    description: Codesystem URL for identifiers.
+                    enum: [https://tools.ietf.org/html/rfc4122]
+                  value:
+                    type: string
+                    description: The X-Request ID for the message.
+                    example: 164996aa-29eb-4f14-a5ac-4b556a7baf6e
+              type:
+                type: string
+                description: Denotes that the bundle is a collection of resources. Representing a Prescription, in this case.
+                enum: [collection]
+              entry:
+                type: array
+                description: A collection of resources contained within the Bundle.
+                items:
+                  oneOf:
+                    - $ref: "#/components/schemas/MedicationRequest-Resource"
+                    - $ref: "#/components/schemas/Organisation-Resource"
+                    - $ref: "#/components/schemas/Practitioner-Resource"
+                    - $ref: "#/components/schemas/PractitionerRole-Resource"
+    MedicationRequest-Resource:
+      type: object
+      description: MedicationRequest
+      properties:
+        fullUrl:
```

Comment:
As this is just a URI, which doesn't resolve I don't think it gives us anything?

Reply:
Removed fullUrls.